package.json

```diff
@@ -58,6 +26,11 @@
     "@electron-forge/maker-squirrel": "6.0.0-beta.54",
     "@electron-forge/maker-zip": "6.0.0-beta.54",
     "@electron-forge/plugin-webpack": "6.0.0-beta.54",
+    "@types/electron-devtools-installer": "^2.2.0",
```

Comment:
Can we remove the ^ in version number, so that we would have exact version installed. This can help prevent some unexpected minor version changed, which cause some issue.

After remove all the ^ you may need to run `yarn` to update `yarn.lock` as well